### PR TITLE
Updates table names used in queries

### PR DESCRIPTION
**Pull request by 01painadam**
## Overview

Changes the names of tables referenced in sql queries to reflect the new table names.

### CODE REVIEW

File: api/controllers/countries.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ function getCountryCriticalSites(req, res) {
   const query = `
     WITH csn_species_count AS (
       SELECT COUNT(*) AS csn_species, site_id
-      FROM csn_species_sites
+      FROM populations_sites
```
Comment (simaob) [inline api/controllers/countries.js]: I'm not sure if this csn_species_sites includes population information or just species-level information. Maybe we can have a look at it together. As species and populations are slightly different concepts.

I would also suggest keeping "critical" in the name, as the distinction at site level is between:

- critical sites;
- important biodiversity areas (ibas);

They are all sites but some are "IBA" and others are "Critical"

File: api/controllers/countries.js [hunk 2]
```diff
@@ -84,7 +84,7 @@ function getCountryCriticalSites(req, res) {
       coalesce(protected, 'Unknown') AS protected,
       csc.csn_species AS csn_species,
       total_percentage
-    FROM sites_csn_points s
+    FROM sites_points s
```
Comment (simaob) [inline api/controllers/countries.js]: would suggest: `critical_sites_points`

File: api/controllers/countries.js [hunk 3]
```diff
@@ -112,7 +112,7 @@ function getCountrySpecies(req, res) {
     INNER JOIN species_country sc on sc.species_id = s.species_id
     INNER JOIN countries c on c.country_id = sc.country_id AND
       c.iso3 = '${req.params.iso}'
-    INNER JOIN populations_iba p on p.species_main_id = s.species_id
+    INNER JOIN populations p on p.species_main_id = s.species_id
```
Comment (simaob) [inline api/controllers/countries.js]: As the information in this populations table is actually related to IBAs, maybe keeping IBA would make sense.

File: api/controllers/countries.js (hunk 1 above)
Comment (01painadam) [reply api/controllers/countries.js]: The thinking here was that the `csn_species_sites` links the `populations_iba` and sites tables via `sites_id` and `wpepopid`, respectively.

Comment (01painadam) [reply api/controllers/countries.js]: It links to populations, species and sites... Will rename to `critical_species_sites`

File: api/controllers/countries.js (hunk 2 above)
Comment (01painadam) [reply api/controllers/countries.js]: renamed to `sites_critical`

**Pull request merged**